### DISCUSSION TITLE
certificates and non-transparent proxy

### DISCUSSION THREAD

**Discussion by sei-avershave** [Azure/AKS#3985] [Q&A]
We have a company proxy that's affecting the certificates when trying to reach out to AKS. 
```
x509: certificate signed by unknown authority (possibly because of “crypto/rsa: verification error” while trying to verify candidate authority certificate “ca”)
```
Is there a way around this other than adding the AKS API domain to the proxy or adding signed certificates to the cluster?

**Comment by philwelz** [comment]
Hey, this is intended by design as of the PKI requirements from Kubernetes described [here](https://kubernetes.io/docs/setup/best-practices/certtficates/#how-certificates-are-used-by-your-cluster).

The Azure docs for AKS describing that the API server cert is signed by the cluster CA can be found [here](https://learn.microsoft.com/en-us/azure/aks/certificate-rotation#aks-certificates-certificate-authorities-and-service-accounts)

Using validly signed (CA) certs is not on the AKS roadmap  - [details](https://github.com/Azure/AKS/issues/1066#issuecomment-505511167)
